client/src/app/components/view-application/view-application.component.html

```diff
@@ -76,15 +76,17 @@
             [class.reject-status-text]="responseData.applicantData.status === 'Reject'"
             [class.pending-status-text]="responseData.applicantData.status === 'Pending'" class="status-text">
             {{applicationStatus}}</span></p>
+      <div class="buttons-container">
         @if (responseData.applicantData.status === 'Pending') {
-        <button mat-raised-button color="primary"
-          (click)="applicationDecision(responseData.applicantData.application, projectID, 'Accept')"
-          class="accept-button">Accept</button>
-        }
-        @if (responseData.applicantData.status === 'Pending') {
-        <button mat-raised-button color="warn"
-          (click)="applicationDecision(responseData.applicantData.application, projectID, 'Reject')">Reject</button>
-        }
+          <button mat-raised-button color="primary"
+            (click)="applicationDecision(responseData.applicantData.application, projectID, 'Accept')"
+            class="accept-button">Accept</button>
+          }
+          @if (responseData.applicantData.status === 'Pending') {
+          <button mat-raised-button color="warn"
+            (click)="applicationDecision(responseData.applicantData.application, projectID, 'Reject')">Reject</button>
+          }
```

Comment:
remove an extra tab before this code block

Reply:
fixed whitespace